app/controllers/api/v0/swagger/models/highlight.rb

```diff
@@ -42,25 +42,69 @@ module Api::V0::Swagger::Models::Highlight
     end
   end
 
+  swagger_schema :GetHighlights do
+    key :required, [:source_type]
+    property :page do
+      key :type, :integer
+      key :description, 'The page number'
+    end
+    property :per_page do
+      key :type, :integer
+      key :description, 'The number of highlights to return per page'
```

Comment:
Ditto, could be misread as "per OpenStax page".  "per paginated results page" would be clearer.

Also, is there a default here?  Is pagination on by default?